components/ws-manager-bridge/src/main.ts

```diff
@@ -61,6 +62,8 @@ export const start = async (container: Container) => {
             return bridge;
         }));
 
+        const clusterServiceServer = serveClusterService();
```

Comment:
this should be injected through the DI container, especially because `ClusterServiceServer` will need to access the DB in the very near future.

Reply:
Ok for now I think. I will need to touch it anywhy when I rebase.